collection-scripts/gather

```diff
@@ -197,6 +204,12 @@ if [ "$clusterscoped" == true ] && [ "$odf" == false ]; then
     pids+=($!)
 fi
 
+if [ "$minimal" == true ]; then
+    echo "Collect minimal resource files..."
+    gather_minimal_resources ${BASE_COLLECTION_PATH} &
```

Comment:
```suggestion
    gather_minimal_resources &
```

Reply:
Done

Reply:
It’s fine I guess. But if you look at actual scripts, none of them read this arg. They read the exported BASE_COLLECTION_PATH. 

Anyways, let’s keep it.